packages/grafana-ui/src/types/theme.ts

```diff
@@ -77,10 +77,8 @@ export interface GrafanaThemeCommons {
     md: string;
     lg: string;
   };
-  panelPadding: {
-    horizontal: number;
-    vertical: number;
-  };
+  panelPadding: number;
```

Comment:
Just a side note. This is and was before an inconsistency with the rest of the theme, which uses strings for numeric values. I think we should strive to have numeric values typed as numbers, i.e. see sizing above.

Reply:
yes, agree. This should (is) a number, as it's also used in code calc, would be painful to have to parse that :) but the other size options should also be numbers